Formula/y/yass-cli.rb

```diff
@@ -0,0 +1,70 @@
+class YassCli < Formula
+  desc "Lightweight and efficient, socks/http forward proxy"
+  homepage "https://letshack.info"
+  url "https://github.com/Chilledheart/yass/releases/download/1.13.1/yass-1.13.1.tar.bz2"
+  sha256 "26015d32a6bdc5bf2670b36c8f5fb68f4a324d577e027bc5eef18ae977e00c9f"
+  license "GPL-2.0-only"
+  head "https://github.com/Chilledheart/yass.git", branch: "develop"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Do we need this? Does `brew livecheck yass-cli` return wrong results if we remove it?

Reply:
I tested it when the project name is still yass. Does it change now?

Reply:
```
➜  ~ brew livecheck yass-cli --debug
/opt/homebrew/Library/Homebrew/brew.rb (Formulary::FromNameLoader): loading yass-cli
/opt/homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::NullLoader): loading yass-cli

Formula:          yass-cli
Livecheckable?:   Yes

URL (stable):     https://github.com/Chilledheart/yass/releases/download/1.13.1/yass-1.13.1.tar.bz2
Strategy:         GithubLatest
/usr/bin/env PATH=/opt/homebrew/opt/gh/bin:/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/bin:/opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin:/opt/homebrew/Library/Homebrew/shims/shared:/usr/bin:/bin:/usr/sbin:/sbin HOME=/Users/hky gh auth token --hostname github.com
/usr/bin/env PATH=/opt/homebrew/opt/gh/bin:/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/bin:/opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin:/opt/homebrew/Library/Homebrew/shims/shared:/usr/bin:/bin:/usr/sbin:/sbin HOME=/Users/hky gh auth token --hostname github.com
/usr/bin/env /opt/homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.19-103-g170a7ea\ \(Macintosh\;\ arm64\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 --location https://api.github.com/repos/Chilledheart/yass/releases/latest --header Accept:\ application/vnd.github+json --write-out '
'\%\{http_code\} --header Authorization:\ token\ ****** --header X-GitHub-Api-Version:2022-11-28 --dump-header /private/tmp/github_api_headers20240908-45359-g9syns
URL (strategy):   https://api.github.com/repos/Chilledheart/yass/releases/latest
Regex (strategy): /v?(\d+(?:\.\d+)+)/i

Matched Versions:
1.13.1

yass-cli: 1.13.1 ==> 1.13.1
```

It works well.

Reply:
I'm not sure you understood me -- does `brew livecheck` return the wrong results _if this block is removed_?

I know it works if we have this `livecheck` block, but we shouldn't use this block if it's not needed because GitHub has stricter rate limits for the API calls made by the `:github_latest` strategy.

Reply:
Yes, the output is different somehow.

<img width="861" alt="截屏2024-09-08 20 53 33" src="https://github.com/user-attachments/assets/9a13a25f-5963-4cc4-985d-2b097b6eaa2c">

Reply:
Please notice two things: 
1. The livecheckable field is now "No".
2. the version becomes an integer instead of point-release style.